test/test_dipolarkernel.py

```diff
@@ -399,3 +399,19 @@ def test_nonuniform_r():
     V0 = K@P
     assert np.round(V0,3) == 1
 #=======================================================================
+
+def test_orisel():

```

Comment:
This is not really a complete test. Add another one where you compare the output of `dipolarkernel` to an (more) explicitly constructed orientationally selective kernel.

Reply:
Added tests against values calculated from Mathematica